src/screens/text-to-nft/generate-art/tabs/history/mock-data.ts

```diff
@@ -0,0 +1,285 @@
+import { OrderStatus, PanoramaEnum, StableDiffusionOrder } from 'src/apis/stable-diffusion/types';
+
+// TODO: Remove this later or use for tests
+export const HISTORY_MOCK_DATA: StableDiffusionOrder[] = [
```

Comment:
Actually, you can try running this version of backend https://github.com/madfish-solutions/stable-diffusion-nft-backend/pull/2 locally to avoid using mock data and prepare to the deployment of new backend version as better as possible. I can share Stable Diffusion API token and/or other credentials with you if need.